source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -457,7 +457,7 @@ def findOverlayClasses(self,clsList):
 			parentWindow=winUser.getAncestor(self.windowHandle,winUser.GA_PARENT)
 			if parentWindow and winUser.getClassName(parentWindow)=="Frame Notification Bar":
 				clsList.append(IENotificationBar)
-		if windowClassName.lower().startswith('mscandui'):
+		if windowClassName.lower().startswith('mscandui') or windowClassName in ("Microsoft.IME.CandidateWindow.View","Microsoft.IME.UIManager.CandidateWindow.Host"):

```

Comment:
Please split this as well

```suggestion
		if (
			windowClassName.lower().startswith('mscandui')
			or windowClassName in (
				"Microsoft.IME.CandidateWindow.View",
				"Microsoft.IME.UIManager.CandidateWindow.Host"
		)):
```